internal/model/plugin.go

```diff
@@ -10,6 +10,7 @@ import (
 
 // Plugin represents a Mattermost plugin in the Plugin Marketplace.
 type Plugin struct {
+	RepoName        string                    `json:"repo_name"`
```

Comment:
1/5 that this filed should be after `Signature` as `HomepageURL` and  `IconData` are directly from the manifest